ci/e2e/oidc_configuration/cluster_with_managed_oidc_config/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable token {
+  type = string
+  sensitive = true
+}
+
+variable url {
+    type = string
+    default = "https://api.stage.openshift.com"
+}
+
+variable operator_role_prefix {
+    type = string
+}
+
+variable account_role_prefix {
+    type = string
+    default = ""
+}
+
+variable cluster_name {
+    type = string
+}
+
+variable aws_region {
+    type = string
+    default = "us-east-1"
+}
+
+variable aws_availability_zones {
+    type      = list(string)
+    default = ["us-east-1a"]
+}
+
+variable replicas {
+    type = string
+    default = "3"
+}
+
+variable openshift_version {
+    type = string
+    default = "4.13.0"
```

Comment:
Also need to be configured by the CI, we would like to prevent changing the code for this

Reply:
It is configured by tfvars.
Do you mean remove the default value?

Reply:
Yes, you right, it configurable.
The CI should always set the version? if yes, I think it better to remove it to have better indication if the CI missed it.
Same for "channel group" below

Reply:
Good point, so I would prefer validate all the variables. WDYT @AsherShoshan if I remove all defaults from variables.tf?

Reply:
the pre-submit check fails without the defaults(

Reply:
As decided - we would try to not using defaults values